Added world IDs to world.h and fixed some typos

Timtech opened the pull request:
My second pull request with some fixes + ID additions

Commit:
Merge pull request #1 from jaller200/master

Merge

Commit:
Update World.h

Added all current IDs

Commit:
Update WorldLoop.cpp

Fixed typos

ghost commented:
Great. You'll want to add the new IDs to the World.cpp file. I'll merge after you do that.

Commit:
added in the IDs from world.h

ghost commented:
Done :)

Commit:
fixed typos

Knightoffaith commented on Source/LUNI Latest Dev/Source/CharactersLoop.cpp:
```diff
@@ -145,7 +145,7 @@ void CharactersLoop(CONNECT_INFO* cfg, Ref< UsersPool > OnlineUsers, Ref< CrossT
 									else if ( usr->nextcid == 1534792735 ) s << " GruntMonkey!\n";
 									else if ( usr->nextcid == 1457240027 ) s << " Shafantastic!\n";
-									else s << " Unknow cid: " << usr->nextcid << endl;
+									else s << "Unknown ID: " << usr->nextcid << endl;
```
One last thing. I would make this Unknown Character ID to stay consistent.

ghost commented:
Not sure what you mean, but feel free to pull + fix it.

ghost commented:
Never mind. Fixing now.

Commit:
fixed with knightoffaith's suggestion

Commit:
Added LEGO Universe Rebuilt Packet Scout

This was made a while ago... changes it to English with some other minor
changes.
